classes/class-wc-connect-shipping-label.php

```diff
@@ -73,11 +73,32 @@ protected function get_packaging_data( WC_Order $order ) {
 			return $packages;
 		}
 
+		protected function get_selected_rates( WC_Order $order ) {
+			$shipping_method = reset( $order->get_shipping_methods() );
```

Comment:
I get a `PHP Strict Standards:  Only variables should be passed by reference in php shell code` from this line.

Can you accomplish this same logic without using `reset()` or by storing the shipping methods in a variable first?